src/main/java/net/consensys/orion/http/handler/send/SendRequest.java

```diff
@@ -53,6 +60,13 @@ public SendRequest(
     this(decodePayload(payload), from, to);
   }
 
+  @JsonAnySetter
```

Comment:
Please use a dedicated `@JsonSetter("privacyGroupId")` if it is possible. It is much cleaner.

Reply:
In fact you should create another constructor with a `@JsonProperty("privacyGroupId")` instead of `@JsonProperty("to")`.

I am also tempted to make `to` an `Optional` here because you will either get the `to` array or a `privacyGroupId`

Reply:
I am not entirely sure about making `to` optional as - if there is a case where they pass `null` aa a third parameter, it will conflict when trying to resolve to either `String` or `String[]`.